OpenEdXMobile/src/main/java/org/edx/mobile/view/adapters/CourseHomeAdapter.kt

```diff
@@ -23,10 +22,12 @@ class CourseHomeAdapter(
     private val itemClickListener: OnItemClickListener
 ) : ListAdapter<SectionRow, RecyclerView.ViewHolder>(SectionRow.SectionRowComparator) {
 
-    private var otherSectionVisible = false
+    private var isAnySectionVisible = false
 
     private var checkedItem = Pair(-1, -1)
 
+    private val sectionVisibility = mutableMapOf<String, Boolean>()
```

Comment:
same here, `expendedSections`

Reply:
It won't work for it. Because the map also contains false values. It would be appropriate if it was a list of expanded sections.

Reply:
sectionVisibilty sounds more like a boolean object instead of a list or map.

Reply:
Updated to `sectionsExpandedState`